libs/gi/db/src/Database/DataManagers/TeamDataManager.ts

```diff
@@ -219,15 +219,18 @@ export class TeamDataManager extends DataManager<
     }
   }
   import(data: object): string {
-    const { teamChars, ...rest } = data as Team & { teamChars: object[] }
+    const { loadoutData, ...rest } = data as Team & { loadoutData: object[] }
     const id = this.generateKey()
     if (
       !this.set(id, {
         ...rest,
         name: `${rest.name ?? ''} (Imported)`,
-        loadoutData: teamChars.map((obj) => ({
-          teamCharId: this.database.teamChars.import(obj),
-        })),
+        loadoutData: loadoutData.map(
+          (obj) =>
+            obj && {
```

Comment:
won't this produce `false` if the obj doesn't exist or something?

Reply:
No it won't. 
![image](https://github.com/frzyc/genshin-optimizer/assets/1754901/26045ea9-5aae-42e1-ad00-e173db3bb068)